src/FunFair.CodeAnalysis.Tests/ProhibitedMethodsDiagnosticsAnalyzerTests.cs

```diff
@@ -401,5 +401,31 @@ void Test(FunFair.Common.Data.ISqlServerDatabase sqlServerDatabase, string sql)
 
             return this.VerifyCSharpDiagnosticAsync(source: test, expected);
         }
+
+        [Fact]
+        public Task RemoteIpAddressIsBannedAsync()
```

Comment:
Can you do a test too where you've got using 

System.Net.*
and all the AspNetCore.* things

too